app/views/assessments/install_assessment.html.erb

```diff
@@ -1,5 +1,29 @@
 <% @title = "Install Assessment" %>
 
+<% content_for :javascripts do %>
+  <script>
+    $(function() {
+      $("#importAll").click(function(e) {
+        const msg = "Are you sure you want to import all assessments?";
+        if (!window.confirm(msg))
+          return false;
+
+        const btn = $(e.target);
+        btn.text("Importing...");
+        btn.addClass("disabled");
+
+        Promise.all(
+            $(".import").map((i, e) => $.post(e.href))
+          )
+          .then(() => {
+            btn.text("Done!");
+            location.reload();
```

Comment:
Any easy way to add some sort of visual indicator that the imports finished successfully? Gonna assume it's hard to use a flash, but the "Done!" shows up for ~100ms so it's pretty hard to notice. If it's not easy to do, nbd

Reply:
Added alert when imports finish